src/components/Meetings.js

```diff
@@ -34,115 +57,113 @@ const useStyles = makeStyles((theme) => ({
   },
 }))
 
-// TODO: Maybe this branching can be reduced?
-export default function Meetings(props) {
-  const classes = useStyles()
-  const { meetings } = props
+const NoMeeting = ({ classes }) => {
+  return (
+    <div className={classes.meetBorder}>
+      <Typography variant='body1' className={classes.meetNoLink} tabIndex='0'>
+        N/A
+      </Typography>
+      <Typography variant='body1' className={classes.meet} tabIndex='0'>
+        N/A
+      </Typography>
+      <Typography variant='body1' className={classes.meet} tabIndex='0'>
+        N/A
+      </Typography>
+      <Typography variant='body1' className={classes.meet} tabIndex='0'>
```

Comment:
I don't know about having `N/A` displayed 5 times in a column when no meetings are found. Maybe you could just display a certain material icon and under it say "No meeting data available". I'd like to see what the others think as well.

Reply:
@amgoodfellow @kdjohnson @nickwoodle @richast thoughts?

Reply:
I could just display a message, something like "No meeting data to display"

Reply:
Personally, I like the _No meeting data to display` option

Reply:
I'll do that then